schemas/apiDefinition.swagger.schema.json

```diff
@@ -2179,6 +2233,19 @@
         }
       }
     },
+    "DynamicOperationCall-parameters-static": {
+      "description": "An object that provides the input parameters to invoke the operation. Each property is the name of a parameter on the target operation.",
+      "type": "object",
+      "additionalProperties": {
+        "description": "The literal value to be used for the input parameter.",
+        "type": [
```

Comment:
Technically, static value can be an object also. If parameter's object doesn't match to parameter's reference definition it is considered as static value to be passed.

Reply:
thanks for clarifying this. I've adjusted the schema accordingly.